godot-kotlin/godot-library/build.gradle.kts

```diff
@@ -1,5 +1,7 @@
 import org.jetbrains.kotlin.gradle.plugin.mpp.KotlinNativeTarget
 
+val platform: String? by project
```

Comment:
This will have an effect with publishing, as the `kotlinMultiplatform` publication will not contain information about the other platforms. I would suggest using the compile task for the specific platform, i.e `compileKotlinLinux` for linux instead of this platform parameter.

```
arguments: godot-library:${{ matrix.task }} -PignoreSamples
```